specs/protocol/README.md

```diff
@@ -607,6 +596,48 @@ This table enumerates the structure of `PrivateData`
 }
 ```
 
+
+### `OrderInstructions`
+> PFI -> Alice: "Here's how to pay us, and how to let us pay you."
+
+| field    | data type                                   | required | description                                                                                                   |
+| -------- | ------------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------- |
+| `payin`  | [`PaymentInstruction`](#PaymentInstruction) | Y        | Object that describes how to pay the PFI (e.g. BTC address, payment link)     |
+| `payout` | [`PaymentInstruction`](#PaymentInstruction) | Y        | Object that describes how be paid by the PFI (e.g. BTC address, payment link) |
+
+
+#### `PaymentInstruction`
+| field         | data type | required | description                                                               |
+| ------------- | --------- | -------- | ------------------------------------------------------------------------- |
+| `link`        | String    | N        | Link to allow Alice to pay PFI, or be paid by the PFI                     |
+| `instruction` | String    | N        | Instruction on how Alice can pay PFI, or how Alice can be paid by the PFI |
+
+
+#### Example OrderInstructions
+```json
+{
+  "metadata": {
+    "from": "did:ex:pfi",
+    "to": "did:key:z6MkoreC1URMQEVKm8wxzhVFntLzvPXYbtH8CEFNKUcSkLWT",
+    "exchangeId": "rfq_01ha83f661fs2avj6qgdhxpg28",
+    "kind": "orderinstructions",
+    "id": "orderinstructions_01ha83f663e3e88fshb06h6g78",
+    "createdAt": "2023-09-13T20:24:37.315Z",
+    "protocol": "1.0"
```

Comment:
will the protocol version be increased? if so, we wouldn't have an OrderInstructions message with 1.0 would we?

Reply:
hmm, we've certainly made quite a few changes to the protocol, but i wasn't part of the versioning discussion when this field first got added here so i will have @mistermoe / @frankhinek chime in here

Reply:
https://github.com/TBD54566975/tbdex/issues/360